src/App.jsx

```diff
@@ -78,6 +79,14 @@ class App extends React.PureComponent {
               currentUser={currentUser}
               removeCurrentUser={this.removeCurrentUser}
             />
+            <AuthenticatedRoute
+              exact
+              path="/newPage"
```

Comment:
I think that the URL should lead to the dashboard page which will contain the Partners stats card component.

Reply:
@FelixWambiri This should change once the dashboard page is ready. It was for testing